src/components/data_submission/NIHAdministrativeInformation.js

```diff
@@ -22,13 +23,16 @@ export const NIHAdministrativeInformation = (props) => {
     const institution = institutions.find((inst) => inst.id === id);
 
     return {
-      displayText: institution.name || 'Unknown',
+      displayText: institution?.name || 'Unknown',
       id: id,
     };
   };
 
   return div({
-    isRendered: [YES_NHGRI_YES_PHS_ID, YES_NHGRI_NO_PHS_ID, NO_NHGRI_YES_ANVIL].includes(formData.nihAnvilUse),
+    isRendered:
+      (studyEditMode ?
+        ['yes_nhgri_yes_phs_id', 'yes_nhgri_no_phs_id', 'no_nhgri_yes_anvil'].includes(formData.properties.nihAnvilUse)
+        : [YES_NHGRI_YES_PHS_ID, YES_NHGRI_NO_PHS_ID, NO_NHGRI_YES_ANVIL].includes(formData.nihAnvilUse)),
```

Comment:
```suggestion
        ['yes_nhgri_yes_phs_id', 'yes_nhgri_no_phs_id', 'no_nhgri_yes_anvil'].includes(formData.properties.nihAnvilUse)
        : [YES_NHGRI_YES_PHS_ID, YES_NHGRI_NO_PHS_ID, NO_NHGRI_YES_ANVIL].includes(formData.nihAnvilUse)),
```

I believe there's four possible states for the `nihAnvilUse`, is this expected?

Reply:
This section is only rendered from 3/4 states.